Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event {
+    var title : String {get}
+    var time : Date {get}
+    var eventDescription: String {get} //NOT related to event content (e.g. workshop content)
+    var colour : String {get}
```

Comment:
Nit: very sorry to say that we're using American English in this repo (so `color` 😢)

Reply:
Damn - I guess it was just wishful thinking 😢

Reply:
Not a blocker but is there a type we can use to better represent a colour as well? Aware that we're using strings elsewhere but we could nip these magic strings in the bud before they get everywhere. Something like [UIKit's `UIColor`](https://developer.apple.com/documentation/uikit/uicolor)